server/middleware/setUpWebSecurity.ts

```diff
@@ -19,19 +19,22 @@ export default function setUpWebSecurity(): Router {
       contentSecurityPolicy: {
         directives: {
           defaultSrc: ["'self'"],
-          // Hash allows inline script pulled in from https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/template.njk
```

Comment:
The comment about the hash, and the hash itself are outdated so I have removed them. The script in the template that it refers to has since been changed to use the nonce value, so it doent need this anymore (and the hash will be different anyway) https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/template.njk#L31

I've also re-ordered the values slightly as well - self, then the nonce value, then external hosts